pom.xml

```diff
@@ -26,7 +26,6 @@
         <hpi.compatibleSinceVersion>2.2.0</hpi.compatibleSinceVersion>
         <java.level>8</java.level>
         <jenkins.version>2.150.3</jenkins.version>
-        <scm-api.version>2.6.3</scm-api.version>
         <hamcrest.version>2.2</hamcrest.version>
```

Comment:
It is possible this is obsolete, too. Check whether you can delete all mention of Hamcrest.